.github/workflows/continuous-integration-workflow.yml

```
@@ -0,0 +1,75 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+  schedule:
+    - cron:  '0 5 * * FRI'
```

```suggestion
  schedule:
    - cron:  '0 5 * * FRI'
```
Any reason for doing a scheduled run?

---

Nothing in particular, just so get a test run at least once a week in case the containers changed.

---

Removed.